ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -335,6 +367,14 @@ Apply Custom Manifest in DataScienceCluster CustomResource Using Test Variables
          END
     END
 
+Apply Custom Manifest in DataScienceCluster CustomResource Using Custom Configuration
+    [Documentation]    Apply custom manifests to a DSC file
+    Log To Console    Applying Custom Manifests
```

Comment:
it is not applying actually

Reply:
Renamed to applying devFlags

Reply:
that wouldn't change much, it would still not being "applying" anything on the cluster

Reply:
what do you suggest?

Reply:
I think that if a keyword is just editing a local YAML file, without actually applying it with OC client. it'd better to make it clear in the kw name like "Generate DataScianceCluster YAML" which gather any edit you need. Or if you need/prefer single keywords, `Generate CustomManifest in DSC YAML` or similar. 

Then I'd expect to find a keyword which actually Applies the DSC resource in the cluster

Reply:
Renamed keyword to Generate CustomManifest in DSC YAML

Reply:
ty, however the same applies to the other keywords in the PR which says "Apply" but doesn-t apply